src/common/pf_lldp.c

```diff
@@ -1020,6 +1020,108 @@ bool pf_lldp_is_peer_info_received (pnet_t * net, uint32_t * timestamp_10ms)
    return is_received;
 }
 
+static void pf_lldp_send_remote_mismatch_alarm (
```

Comment:
I think this operation should be placed in pdport which handles the pdport data checks. Operation name could be "pf_pdport_peer_indication(). Operation called when peer info is changed. Nice if lldp do not need to be aware of alarms and mapping of ports to DAP

Reply:
Done